### ISSUE TITLE
[Model] getting error while running HINT 

### ISSUE THREAD

**Issue by iamyihwa** [Nixtla/neuralforecast#659]
### What happened + What you expected to happen

Hi, I am following the example from [hierarchical forecast](https://nixtla.github.io/neuralforecast/examples/hierarchicalnetworks.html)

Getting error that says Attribute Error: 'HINT' object has no attribute 'early_stop_patience_steps'
![image](https://github.com/Nixtla/neuralforecast/assets/12494156/c4ef2d72-8faf-45a3-a833-e71f41ce95dc)

tried both with this line 12 commented, and not commented, but getting the same error 

![image](https://github.com/Nixtla/neuralforecast/assets/12494156/994e8eda-23d3-4d60-8378-b8c74ced4fdd)

### Versions / Dependencies

1.5.0 
(installed via !pip install git+https://github.com/Nixtla/neuralforecast.git  ) 

### Reproduction script

# Horizon and quantiles
level = np.arange(0, 100, 2)
qs = [[50-lv/2, 50+lv/2] if lv!=0 else [50] for lv in level]
quantiles = np.sort(np.concatenate(qs)/100)

# HINT := BaseNetwork + Distribution + Reconciliation
nhits = NHITS(h=horizon,
              input_size=24,
              loss=GMM(n_components=10, quantiles=quantiles),
              hist_exog_list=['month'],
              max_steps=2000,
              early_stop_patience_steps=10,   
              val_check_steps=50,
              scaler_type='robust',
              learning_rate=1e-3,
              valid_loss=sCRPS(quantiles=quantiles))

model = HINT(h=horizon, S=S_df.values,
             model=nhits,  reconciliation='BottomUp')

Y_df['y'] = Y_df['y'] * (Y_df['y'] > 0)
nf = NeuralForecast(models=[model], freq='MS')
# Y_hat_df = nf.cross_validation(df=Y_df, val_size=12, n_windows=1)
nf.fit(df=Y_train_df, val_size=12)
Y_hat_df = nf.predict()

### Issue Severity

None
